조은채/BOJ/S2_1927.java

```diff
@@ -0,0 +1,29 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.PriorityQueue;
+
+public class S2_1927 {
+    public static void main(String[] args) throws IOException {
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+
+        int N =  Integer.parseInt(br.readLine());
+
+        PriorityQueue<Integer> pq = new PriorityQueue<>();
+
+        for (int i = 0; i < N; i++) {
+            int x = Integer.parseInt(br.readLine());
+
+
+            if (x == 0){
+                if (pq.size() == 0){
```

Comment:
```suggestion
                if (pq.isEmpty()){
```

PriorityQueue 는 isEmpty() 를 아래와 같이 구현하고 있으니 제공되는 API를 사용하는게 일관성을 위해서는 좋을거라 생각 됩니다.

```java
return size() == 0;
```